src/ForgotPasswordController.js

```diff
@@ -72,8 +72,7 @@ function (Okta, FormController, Enums, FormType, ValidationUtil, Util, ContactSu
       },
       save: function () {
         var self = this;
-        var urlParams = Util.getJsonFromUrl(window.location.search);
-        var relayState = urlParams.fromURI || '';
+        var relayState = this.settings.get('relayState');
```

Comment:
this breaks the previous functionality right? I'm guessing the settings param is preferred over the url, but we should still have a criteria here to chose the relayState in that order
1 if in settings, use it
2 not in settings, use the one from url
3 no relayState? then empty string

Or is there a reason we don't want to support the url relayState anymore?

Reply:
okay... but then more to the point of *not* removing the functionality that already existed right? Unless as I was asking that is the intention

Reply:
This piece of code was added 14 days back and is not an existing functionality in any widget release.

Reply:
Yes, as @santhoshbalakrishnan mentioned the code to get fromURI form `window.location.search` was added recently but after recent discussions this change was preferred.